recordedit/recordEdit.app.js

```diff
@@ -318,6 +321,7 @@
                 }
             });
         } catch (exception) {
```

Comment:
This is a universal try-catch for this app. Should we instead wrap the code in `Utils.catchWrapper`?